examples/get_transactions.py

```diff
@@ -0,0 +1,73 @@
+import os
+import sys
+import csv
+import getpass
+import StringIO
+import argparse
+
+from collections import defaultdict
+
+# Fix for CERTIFICATE_VERIFY_FAILED error in python 2.7.9 (see https://bugs.python.org/issue22417)
+import ssl
+if hasattr(ssl, '_create_unverified_context'):
+    ssl._create_default_https_context = ssl._create_unverified_context
+
+# TODO: fix this
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
```

Comment:
This can be omitted. The script can be run as follows from the root of the project, e.g.:

    $ python examples/get_transactions.py 2014

In case the `PYTHONPATH` is not set, you can run the following instead:

    $ PYTHONPATH=. python examples/get_transactions.py 2014